src/CandidateRankingParser.php

```diff
@@ -9,6 +9,10 @@ class CandidateRankingParser
 {
     private const ORDERED_DELIM = ">";
     private const EQUAL_DELIM = "=";
```

Comment:
Do you like the prohibition of these characters from text that should be parsed into a CandidateRanking or a Ballot? I'm trying to prevent someone from passing `100*A>B>C` into `CandidateRankingParser` or `BallotParser` and winding up with Candidates `100*A`, `B`,`C`, when what they really want in that situation is to pass the same text to `NBallotParser`.